package.json

```diff
@@ -34,18 +34,50 @@
     "test-links": "markdown-link-check"
   },
   "devDependencies": {
+    "@babel/core": "^7.8.0",
+    "@babel/preset-env": "^7.8.0",
+    "@babel/preset-react": "^7.8.0",
+    "@babel/preset-typescript": "^7.8.0",
     "@fluentui/token-pipeline": "^0.3.11",
     "@rnx-kit/dep-check": "^1.5.10",
+    "babel-jest": "^24.9.0",
+    "babel-plugin-module-resolver": "^4.0.0",
     "beachball": "^1.35.0",
     "lage": ">=0.17.4 <1.0.0",
     "markdown-link-check": "^3.8.7",
-    "react-dom": "^16.13.1"
+    "metro-react-native-babel-preset": "^0.66.0",
+    "react-dom": "17.0.1"
   },
   "workspaces": {
     "packages": [
       "apps/*",
       "packages/**",
       "scripts"
     ]
+  },
+  "resolutions": {
+    "@office-iss/react-native-win32": "^0.64.8",
+    "metro": "^0.66.2",
+    "metro-babel-register": "^0.66.2",
+    "metro-babel-transformer": "^0.66.2",
+    "metro-cache": "^0.66.2",
+    "metro-cache-key": "^0.66.2",
+    "metro-config": "^0.66.2",
+    "metro-core": "^0.66.2",
+    "metro-hermes-compiler": "^0.66.2",
+    "metro-inspector-proxy": "^0.66.2",
+    "metro-minify-uglify": "^0.66.2",
+    "metro-react-native-babel-preset": "^0.66.2",
+    "metro-react-native-babel-transformer": "^0.66.2",
+    "metro-resolver": "^0.66.2",
+    "metro-runtime": "^0.66.2",
+    "metro-source-map": "^0.66.2",
+    "metro-symbolicate": "^0.66.2",
+    "metro-transform-plugins": "^0.66.2",
+    "metro-transform-worker": "^0.66.2",
+    "react": "17.0.1",
+    "react-native": "^0.64.3",
+    "react-native-windows": "^0.64.3",
+    "react-native-macos": "^0.64.6"
```

Comment:
These are a lot of resolutions at our root. Are these to silence the yarn aggregator warnings? Will it also be easy enough to know when / we to bump these in the future?

Reply:
Ah ok, this is to force the use of Metro 0.66. Could you elaborate more on why we want that, and maybe when we don't need to have resolutions (aka which version of react native has 0.66 as a minimum)?

Reply:
There are two reasons I did this. The first is to force Metro to be >= 0.66.2 which is required for adopting a bunch of new @rnx-kit updates. react-native 0.64 uses an older build of Metro, but it works fine with 0.66.2.

The second reason is that I wanted alignment of react and all the react-native packages across all dependencies. It had the side-effect of silencing the warnings, but the goal was to avoid problems from multiple versions of the same package.

Reply:
So I had put these in early on, and decided to try removing the react & react-native. Turns out I was able to do it, leaving only the metro resolutions (which are definitely required).

Reply:
Yeah, the monorepo gets it's react / react-native definitions from the test apps (the only things that have them as a hard dependency). In general, I like the resolutions at root as a better place to specify. It's not at all obvious that the test apps decide which react dependencies are actually installed.